app/src/main/java/protect/card_locker/LoyaltyCardEditActivity.java

```diff
@@ -562,10 +569,15 @@ public void onActivityResult(ActivityResult result) {
             skipOnce = savedInstanceState.getInt("skipOnce") == 1 ? true : false;
         }
         mCropperOptions = new UCrop.Options();
-        mCropperOptions.setCompressionFormat(Bitmap.CompressFormat.JPEG);
+        setCropperOptions();
         setCropperTheme();
     }
 
+    private void setCropperOptions(){
+        mCropperOptions.setCompressionFormat(Bitmap.CompressFormat.JPEG);
+        mCropperOptions.setFreeStyleCropEnabled(false);
```

Comment:
>Enables freely changing the Aspect Ratio>

Personally I find free style crop make it a little janky because the crop area would prevent user from two finger pinching/rotating the image

Do toggle it around and test it however